compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.desktop.kt

```diff
@@ -667,6 +668,9 @@ internal class ComposeSceneMediator(
         private val _accessibilityControllers = linkedMapOf<SemanticsOwner, AccessibilityController>()
         val accessibilityControllers get() = _accessibilityControllers.values.reversed()
 
+        val semanticsOwners: Collection<SemanticsOwner>
```

Comment:
What if we make it `mutableStateListOf`? Otherwise usage of it is limited

Reply:
Good idea.

Reply:
Done